docs/pages/cli/verify.mdx

```diff
@@ -65,3 +70,31 @@ Otherwise, the generated bytecode is different and therefore verification fails.
   export ETHERSCAN_API_KEY= ** blockscout API key goes here **
   pnpm mud verify --rpc $HOLESKY_RPC --worldAddress $WORLD_ADDRESS --verifier blockscout --verifier-url https://eth-holesky.blockscout.com/api
   ```
+
+## Verifying library contracts
```

Comment:
Rephrasing slightly because this works for any manual contract verification and "library contracts" might be confusing (since there is also a "library" concept in solidity but that's not what we mean here)

```suggestion
## Manual contract verification
```